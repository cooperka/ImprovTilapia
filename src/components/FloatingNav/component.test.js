import React from 'react';
import renderer from 'react-test-renderer';

import Component from './component';

it('renders as expected', async () => {
  const tree = renderer.create(<Component />);
  expect(tree.toJSON()).toMatchSnapshot();
});
