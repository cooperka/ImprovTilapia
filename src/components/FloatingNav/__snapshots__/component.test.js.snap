// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders as expected 1`] = `
<View
  style={
    Object {
      "left": 16,
      "position": "absolute",
      "top": 36,
    }
  }
>
  <Text />
</View>
`;
